bio/kraken2/build/wrapper.py

```
@@ -0,0 +1,24 @@
+__copyright__ = "Copyright 2026, Alessandro Leone"
+__email__ = "alessandro.leone@unito.it"
+__license__ = "MIT"
+import os
+from snakemake.shell import shell
+
+extra = snakemake.params.get("extra", "")
+
+assert snakemake.output.get("hash") is not None, "Output -> hash is required"
+assert snakemake.output.get("opts") is not None, "Output -> opts is required"
+assert snakemake.output.get("taxo") is not None, "Output -> taxo is required"
+db_dir = os.path.abspath(snakemake.input.db)
```

Ruff (0.15.7)

[error] Undefined name `snakemake`

(F821)